docs/meta.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Functional",
+  "type": "Library",
+  "support": {
```

Comment:
@MykolaGolubyev originally i had `viewOn` here and i'd keep it if there is a way to disable it on some pages. For example, i don't want this link on the landing `index.md` page but i'd like to have it for all the files inside of `reference-manual`. Is there a way to do this? I could not find it. Thanks

Reply:
just in case - viewOn should point to original md file that is the source for the page. 
What makes you want to hide it for some pages?

Reply:
It generates urls automatically.. for the index.md it generates `url//index.` instead of `url/index.md`. If all urls would be correct, maybe i'd keep `viewOn` as is

Reply:
this looks like a bug! could you please submit another issue?

Reply:
@MykolaGolubyev all ready for you: https://github.com/testingisdocumenting/znai/issues/1210